src/Extension/FluentExtension.php

```diff
@@ -835,4 +846,106 @@ public function cacheKeyComponent()
+        $currentLocale = Locale::getCurrentLocale();
+        $locale = $currentLocale->Locale;
+        $language = $currentLocale ? strtok($locale, '_') : '';
+
+        foreach ($translated as $table => $translatedFields) {
+            foreach ($translatedFields as $translatedField) {
+                // Find field matching this translated field
+                // If the translated field has an ID suffix also check for the non-suffixed version
+                // E.g. UploadField()
+                $field = $fields->dataFieldByName($translatedField);
+                if (!$field && preg_match('/^(?<field>\w+)ID$/', $translatedField, $matches)) {
+                    $field = $fields->dataFieldByName($matches['field']);
+                }
+
+                // Highlight any translatable field
+                if ($field && !$field->hasClass('LocalisedField')) {
+                    // Add a language indicator next to the fluent icon
+                    $title = $field->Title();
+
+                    $titleClasses = 'fluent-locale-label';
+                    $modifiedTitle = 'Using default locale value';
+
+                    $field->setTitle(
+                        DBHTMLText::create()->setValue(
+                            sprintf(
+                                '<span class="%s" title="%s">%s</span>%s',
+                                $titleClasses,
+                                $modifiedTitle,
+                                $language,
+                                $title
+                            )
+                        )
+                    );
+
+                    $field->addExtraClass('LocalisedField');
+                }
+            }
+        }
+
+        $this->addLocaleIndicatorMessage($fields);
+    }
+
+    /**
+     * Adds a UI message to indicate whether you're editing in the default locale or not
+     *
+     * @param  FieldList $fields
+     * @return $this
+     */
+    protected function addLocaleIndicatorMessage(FieldList $fields)
+    {
+        if (Config::inst()->get(__CLASS__, 'disable_current_locale_message')) {
+            return $this;
+        }
+
+        // If the field is already present, don't add it a second time
+        if ($fields->fieldByName('CurrentLocaleMessage')) {
+            return $this;
+        }
+
+        $localeNames     = Locale::getCached()->map('Locale', 'Title')->toArray();
+        $defaultLocale   = Locale::getDefault();
+        $isDefaultLocale = ($defaultLocale === Locale::getCurrentLocale());
+        $defaultLocaleKey = $defaultLocale ? $defaultLocale->Locale : '';
+        $currentLocale = Locale::getCurrentLocale();
+        $currentLocaleKey = $currentLocale ? $currentLocale->Locale : '';
+        $messageClass    = ($isDefaultLocale) ? 'alert-success' : 'alert-info';
+        $message         = ($isDefaultLocale)
+            ? _t(__CLASS__.'DefaultLocale', 'This is the default locale')
+            : _t(
+                __CLASS__.'DefaultLocaleIs',
```

Comment:
Missing `.` before DefaultLocaleIs - this will evaluate to `TractorCow\Fluent\.....\...ExtensionDefaultLocale` - could be why you're seeing weird php. prefixes in the en.yml file

Reply:
Of course, you are correct. That is exactly the reason why.